src/mockServer/mockServer.js

```diff
@@ -38,6 +41,7 @@ const setupApi = ({
     urlPrefix: apiHost,
     models: {
       appDataResource: Model,
+      appActionsResource: Model,
```

Comment:
Actually you can call it `appAction`, it's just  that miragejs automatically apply plural on the name, and `data` is an exception which doesn't work. But for `action` it will work well.